06a-WGS.Rmd

```diff
@@ -0,0 +1,118 @@
+- [MuSE](https://bioinformatics.mdanderson.org/public-software/muse/) This is a beneficial mutation calling tool when you have both tumor and normal datasets. It uses a Markov Substitution model for Evolution, which models the evolution of the allelic composition of the tumor and normal tissue at each reference base. 
+- [SvABA](https://github.com/walaj/svaba) This tool is especially useful for calling insertions and deletions (indels) because it utilizes a String Graph Assembler to assemble aberrantly aligned sequence reads that reflect indels or structural variants. Indels can be difficult to detect with standard alignment-based variant callers. 
+- [Strelka](https://github.com/Illumina/strelka) This is a small variant caller designed by illumina. It is used for identifying germline variants in cohorts of samples and somatic variants in tumor/normal sample pairs. 
+- [SomaticSniper](https://gmt.genome.wustl.edu/packages/somatic-sniper/) SomaticSniper can be used to identify SNPs in tumor/normal pairs. It calculates the probability that the tumor and normal genotypes are different and reports this probability as a somatic score. 
+- [Pindel](https://www.sanger.ac.uk/tool/pindel/) Pindel is a tool that uses a pattern growth approach to detect breakpoints of large deletions, medium size insertsion/inversion, tandem duplications. 
+
+Here are commonly used variant calling annotation tools: 
+- [Annovar](https://annovar.openbioinformatics.org/en/latest/) This is a helpful tool for annotating, filtering, and combining the output data from the above tools. It can be used for gene-based, region-based, or filter-based annotations. 
+- [GENCODE](https://www.gencodegenes.org/#) This tool can be used to identify and classify gene features in human and mouse genomes. 
+- [dbSNP](https://www.ncbi.nlm.nih.gov/snp/) This is a resource to look up specific human single nucleotide variations, microsatellites, and small-scale insertions and deletions. 
+- [Ensembl](https://uswest.ensembl.org/index.html) This resource is a genome browser for annotating genes from a wide variety of species. 
+
+These are some of the commonly used copy number variation analysis tools: 
+- [GATK](https://gatk.broadinstitute.org/hc/en-us/articles/360035531092--How-to-part-I-Sensitively-detect-copy-ratio-alterations-and-allelic-segments) GATK has a variety of tools that ca nbe used to study changes in copy numbers of genes. This link provides a tutorial for how to use the tools. 
+- [AscatNGS](https://www.crick.ac.uk/research/labs/peter-van-loo/software) This tools (allele-specific copy number analysis of tumors) is specific for WGS copy number variation analysis. It can be used to dissect allele-specific copy numbers of tumors by estimating and adjusting for tumor ploidy and nonaberrant cell admixture.  
```

Comment:
```suggestion
- [AscatNGS](https://www.crick.ac.uk/research/labs/peter-van-loo/software) These tools (allele-specific copy number analysis of tumors) are specific for WGS copy number variation analysis. It can be used to dissect allele-specific copy numbers of tumors by estimating and adjusting for tumor ploidy and nonaberrant cell admixture.  
```